.gitignore

```diff
@@ -44,3 +44,4 @@ webapp-django/.env
 # exclude filesystem storage directories
 primaryCrashStore/
 processedCrashStore/
+/tags
```

Comment:
See http://www.peterbe.com/plog/ctags-in-atom-on-osx

This is a common convention file to put in a project root. I just happened to be messing around with this in Atom whilst I was working on this branch. Please forgive :)